Using 8 microsteps for RA/DEC in web configurator (default value) leads to compilation failure

Default microsteps for RA/DEC in OAT Firmware Configuration web V2.15 is 8.

`// Unique ConfigKey: TR:OAT,FWT:L,SL:N,HS:N,BD:M21,RST:N9,RDO:TU,RATP1500:O90:A3000:V1800:S8:T256:,RTR:N,RT:1,DS:N9,DDT:TU,DATP1500:O90:A3000:V1800:S8:T256:,DT:1,DLIN90:D45:,STL:N,DY:LCD,GP:Y,LM:N,FC:N,APT:N,RAH:Y,RAHAP53:S10: `

Compilation fails with this default setup. But it works changing it to 16.

```
2024-03-02 09:10:21,039:ERROR:In file included from .pio\libdeps\mksgenlv21\InterruptStepper\include/Stepper.h:25:0, 
2024-03-02 09:10:21,039:ERROR:                 from src\StepperConfiguration.hpp:13, 
                 from src\Mount.hpp:11, 
                 from src\HallSensorHoming.hpp:5, 
                 from src\Mount.cpp:5: 
.pio\libdeps\mksgenlv21\InterruptStepper\include/AccelerationRamp.h: In instantiation of 'class AccelerationRamp<256, 16000000, 314, 314>': 
.pio\libdeps\mksgenlv21\InterruptStepper\include/Stepper.h:306:65:   required from 'static uint32_t Stepper<INTERRUPT, DRIVER, RAMP>::distanceToGo() [with INTERRUPT = IntervalInterrupt<(Timer)4>; DRIVER = Driver<Pin<60>, Pin<61> >; RAMP = AccelerationRamp<256, 16000000, 314, 314>; uint32_t = long unsigned int]' 
src\InterruptAccelStepper.h:89:37:   required from 'uint32_t InterruptAccelStepper<STEPPER>::distanceToGo() [with STEPPER = Stepper<IntervalInterrupt<(Timer)4>, Driver<Pin<60>, Pin<61> >, AccelerationRamp<256, 16000000, 314, 314> >; uint32_t = long unsigned int]' 
src\Mount.cpp:1428:66:   required from here 
.pio\libdeps\mksgenlv21\InterruptStepper\include/AccelerationRamp.h:91:5: error: static assertion failed: Amount of steps per stair has to be at most 128 
     static_assert(STEPS_PER_STAIR <= 128, "Amount of steps per stair has to be at most 128"); 
     ^~~~~~~~~~~~~ 
.pio\libdeps\mksgenlv21\InterruptStepper\include/AccelerationRamp.h:92:5: error: static assertion failed: Amount of steps per stair has to be power of 2 
     static_assert(is_pow2(STEPS_PER_STAIR), "Amount of steps per stair has to be power of 2"); 
     ^~~~~~~~~~~~~ 
```

Tested in OATFWGUI v1.0.6 (FAILS) and v1.0.7 (FAILS):

- develop repo: FAILS
- Arduino v1.13.0 Pre-Release repo: FAILS
- v1.12.6 Pre-Release repo: FAILS
- v1.11.5: COMPILES

--- a/src/modules/Home.jsx
+++ b/src/modules/Home.jsx
@@ -13,7 +13,7 @@ const Home = (props) => {
         <div className="intro-link-child">
         <h3 className="intro-header">OAT/OAM FIRMWARE CONFIGURATION</h3>
           <a className="intro-link" href="steps">START</a>
-          <p className="version">V2.15</p>
+          <p className="version">V2.16</p>
         </div>
       </div>
 

--- a/src/modules/WizardStep.jsx
+++ b/src/modules/WizardStep.jsx
@@ -20,15 +20,15 @@ const Defaults = {
     HoldPercentage: { BY: 0, N9: 10, N8: 10, N49: 10, N48: 10 },
     Speed: { BY: 400, N9: 1800, N8: 1800, N49: 1200, N48: 1200 },
     Acceleration: { BY: 600, N9: 3000, N8: 3000, N49: 3000, N48: 3000 },
-    RASlewMicrostepping: { BY: 1, N9: 8, N8: 16 },
+    RASlewMicrostepping: { BY: 1, N9: 16, N8: 16 },
     RATrackMicrostepping: { BY: 1, N9: 256, N8: 256 },
-    DECSlewMicrostepping: { BY: 1, N9: 8, N8: 16 },
+    DECSlewMicrostepping: { BY: 1, N9: 16, N8: 16 },
     DECGuideMicrostepping: { BY: 1, N9: 256, N8: 256 },
     FocuserMicrostepping: { BY: 1, N9: 8, N8: 8, N49: 8, N48: 8 },
     AZALTMicrostepping: { BY: 1, N9: 16, N8: 16 },
     OAMSpeed: { N9: 2.0, N8: 2.0, N9O: 2.0, N8O: 2.0 },
     OAMAcceleration: { N9: 2.0, N8: 2.0, N9O: 2.0, N8O: 2.0 },
-    OAMMicrostepping: { N9: 64, N8: 64, N9O: 64, N8O: 64 },
+    OAMMicrostepping: { N9: 128, N8: 128, N9O: 128, N8O: 128 },
     RAHallSensorPin: { OAT: 53, OAM: 27 },
 }
 

